mockup1/assets/76f6034.css

```diff
@@ -453,130 +458,162 @@ ul#container > li::before {
+li. {background-color: #b68bbf}
+li. {background-color: #B8FF92}
+li. {background-color: #dedeea}
+li. {background-color: #ffb4b4}
+li. {background-color: #fef7cd}
+li. {background-color: #d6dae9}
+li. {background-color: #b3a7a5}
+li. {background-color: #bf6972}
+li. {background-color: #76cdf1}
+li. {background-color: #9fc814}
+li. {background-color: #afb0ab}
+li. {background-color: #8dc688}
+li. {background-color: #fa8888}
+li. {background-color: #b87db0}
+li. {background-color: #aaa5a1}
+li. {background-color: #B987B0}
+li. {background-color: #90bcb4}
+li. {background-color: #d6e6e3}
+li. {background-color: #d1f0f5}
+li. {background-color: #f0fef0}
+li. {background-color: #d0c5b1}
+li. {background-color: #cdccfe}
+li. {background-color: #f8c1af}
+li. {background-color: #71c77a}
+li. {background-color: #a0ffff}
+li. {background-color: #56aba2}
+li. {background-color: #edeef3}
+li. {background-color: #7598ad}
+li. {background-color: #d8ffff}
+li. {background-color: #73A4DC}
+li. {background-color: #ffdeff}
+li. {background-color: #c1cf95}
+li. {background-color: #ffeb70}
+li. {background-color: #ffe24f}
+li. {background-color: #cc9759}
+li. {background-color: #d8b5bc}
+li. {background-color: #dfffc9}
+li. {background-color: #70a8fc}
+li. {background-color: #80ffff}
+li. {background-color: #d7a35e}
+li. {background-color: #d5c8ff}
+li. {background-color: #c1b9b8}
+li. {background-color: #dfffc9}
+li. {background-color: #98c1dd}
+li. {background-color: #A6FFFF}
+li. {background-color: #adfb94}
+li. {background-color: #f6cdc8}
+li. {background-color: #7bc2d4}
+li. {background-color: #d4a9ff}
+li. {background-color: #e5ff4d}
+li. {background-color: #b5ffea}
+li. {background-color: #e2e4af}
+li. {background-color: #a4a0d5}
+li. {background-color: #e5665d}
+li. {background-color: #f6e4e2}
+li. {background-color: #e8e4a3}
+li. {background-color: #969b9c}
+li. {background-color: #cdfb0c}
+li. {background-color: #92d2a3}
+li. {background-color: #ff855f}
+li. {background-color: #b5a5a5}
+li. {background-color: #9798e1}
+li. {background-color: #f8d593}
+li. {background-color: #e4a7b4}
+li. {background-color: #fceee5}
+li. {background-color: #d3f894}
+li. {background-color: #fd7575}
+li. {background-color: #94cb76}
+li. {background-color: #95b3d4}
+li. {background-color: #f3f4ef}
+li. {background-color: #82e3fc}
+li. {background-color: #edeee9}
+li. {background-color: #fc8b82}
+li. {background-color: #f4c482}
+li. {background-color: #a7c3aa}
+li. {background-color: #1dd803}
+li. {background-color: #d7d5bc}
+li. {background-color: #12a1f8}
+li. {background-color: #8586c6}
+li. {background-color: #aa7177}
+li. {background-color: #a574ba}
+li. {background-color: #e5deee}
+li. {background-color: #faacdd}
+li. {background-color: #6fc88d}
+li. {background-color: #f4c96f}
+li. {background-color: #5da29b}
+*/
+
+a.trhelp {
+	border: 0 solid #c6d4e0;
+	border-radius: .75rem;
+	padding: .25rem .5rem;
+	color: black;
+	background-color: #ebf0f4;
```

Comment:
The background color makes the "?" barely visible with the way that icon appears on apple devices. See screenshot below
![photo5573597906791933865](https://user-images.githubusercontent.com/1696128/37506847-34c238d6-2905-11e8-9aee-6d7c7f97a43f.jpg)

Reply:
@deniak: I see, you are right. Actually, [that glyph seems too dim on several platforms, even against a white background](https://emojipedia.org/white-question-mark-ornament/). I'm inclined to replace it with a simple question mark in bold, **`?`**. What do you say?

Reply:
yes, that will simplify things!